clients/mobile/src/components/pages/NoVaultPage/NoVaultPage.styled.ts

```
@@ -0,0 +1,15 @@
+import styled from 'styled-components/native';
+import { PageContent } from '~/lib/ui/components/PageContent';
+import { getColor } from '~/lib/ui/utils';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)